examples/amplify-react/src/viewer.js

```diff
@@ -0,0 +1,251 @@
+import { Role } from 'amazon-kinesis-video-streams-webrtc'
+import WebRtcClient from './webRtcClient'
+
+export default class Viewer {
+  constructor(setState, onStatsReport, onRemoteDataMessage) {
+    this.setState = setState
+    this.onStatsReport = onStatsReport
+    this.onRemoteDataMessage = onRemoteDataMessage
+    this.signalingClient = null
+    this.peerConnection = null
+    this.dataChannel = null
+    this.peerConnectionStatsInterval = null
+    this.localStream = null
+    this.remoteStream = null
+    this.localViewSrcObject = null
+    this.remoteViewSrcObject = null
+  }
+
+
+
+  setLocalViewSrcObject(srcObject) {
+    console.log(`setting local view srcObject: ${srcObject}`)
+    this.localViewSrcObject = srcObject
+    this.setState({
+      localViewSrcObject: srcObject
+    })
+  }
+
+  setRemoteViewSrcObject(srcObject) {
+    console.log(`setting remote view srcObject: ${srcObject}`)
+    this.remoteViewSrcObject = srcObject
+    this.setState({
+      remoteViewSrcObject: srcObject
+    })
+  }
+
+  async start(credentials, params) {
+    const {
+      region,
+      endpoint,
+      channelName,
+      clientId,
+      sendAudio,
+      sendVideo,
+      openDataChannel,
+      widescreen,
+      forceTURN,
+      natTraversalDisabled,
+      useTrickleICE,
+    } = params
+
+    this.webRtcClient = new WebRtcClient(
+      Role.VIEWER,
+      credentials,
+      region,
+      endpoint,
+      channelName,
+      clientId,
+      natTraversalDisabled,
+      forceTURN,
+    )
+
+    const channelARN = await this.webRtcClient.getChannelARN()
+    console.log('[VIEWER] Channel ARN: ', channelARN)
+
+    const endpointsByProtocol = await this.webRtcClient.getEndpointsByProtocol()
+    console.log('[VIEWER] Endpoints: ', endpointsByProtocol)
+
+    const iceServers = await this.webRtcClient.getIceServers()
+    console.log('[VIEWER] ICE servers: ', iceServers)
+
+    this.signalingClient = await this.webRtcClient.getSignalingClient()
+
+    const resolution = widescreen ? {
+      width: {
+        ideal: 1280
+      },
+      height: {
+        ideal: 720
+      }
+    } : {
+      width: {
+        ideal: 640
+      },
+      height: {
+        ideal: 480
+      }
+    }
+
+    const constraints = {
+      video: sendVideo ? resolution : false,
+      audio: sendAudio,
+    }
+
+    const configuration = {
+      iceServers,
+      iceTransportPolicy: forceTURN ? 'relay' : 'all',
+    };
+
+    this.peerConnection = new RTCPeerConnection(configuration);
+    if (openDataChannel) {
+      this.dataChannel = this.peerConnection.createDataChannel('kvsDataChannel');
+      this.peerConnection.ondatachannel = event => {
+        event.channel.onmessage = this.onRemoteDataMessage;
+      };
+    }
+
+    // Poll for connection stats
+    this.peerConnectionStatsInterval = setInterval(() => this.peerConnection.getStats().then(this.onStatsReport), 1000);
```

Comment:
Same here. Can we raise it to a 5 second default? Why arent we accepting `peerConnectionStatsInterval` setting here like in master and just defaulting?